local-dev/rabbitmqqueues.bicep

```diff
@@ -125,7 +122,7 @@ output result object = {
     '/planes/kubernetes/local/namespaces/${rabbitmq.metadata.namespace}/providers/apps/Deployment/${rabbitmq.metadata.name}'
   ]
   values: {
-    queue: queue
+    queue: 'queue'
```

Comment:
Was this pointing to an object before and now why is it a string?

Reply:
No it was pointing to string as well. I corrected it here to the right queue name